dev/test-studio/structure/groupByOption.ts

```diff
@@ -0,0 +1,30 @@
+import {StructureBuilder} from 'sanity/lib/dts/src/_exports/desk'
+import {Schema} from '@sanity/types'
+
+type StructureGroup = 'v3' // extend with union strings
+
+export interface StructureGroupOption {
+  structureGroup?: StructureGroup
+}
+
+type MaybeStructureOptions = StructureGroupOption | undefined
+
+export function typesInOptionGroup(
```

Comment:
Let me know if I should delete this. I just prefer defining structure from the schema 😅 ; this is a minimal approach with some typing to make find usage work.